crates/bevy_input/src/touch.rs

```diff
@@ -156,7 +155,7 @@ impl Touches {
     pub fn iter_just_released(&self) -> impl Iterator<Item = &Touch> + '_ {
```

Comment:
we can now use the .values() iterator for these and remove the `+ '_` lifetime annotation

```rust
    pub fn iter_just_released(&self) -> impl Iterator<Item = &Touch> {
        self.just_released.values()
    }
```